controllers/gatekeepersync/gatekeeper_constraint_sync.go

```diff
@@ -98,7 +98,7 @@ func (r *GatekeeperConstraintReconciler) Reconcile(
 		return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil
 	}
 
-	log.Info("Reconciling a Policy with one or more Gatekeeper constraints")
+	log.V(2).Info("Reconciling a Policy with one or more Gatekeeper constraints")
```

Comment:
Nit: other changes here use V(1) for the same kind of "Reconciling policy..." messages.